packages/crashlytics/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## 6.3.0
```

Comment:
Why wasn't crashlytics bumped to v11 JS SDK?

Reply:
There is no JavaScript SDK for Crashlytics.

Reply:
I know, it is still in the package.json and the peer dependencies.

https://github.com/capawesome-team/capacitor-firebase/blob/main/packages/crashlytics/package.json#L67

Reply:
This is a dev dependency, not a peer dependency.

Reply:
Sorry, you are right